src/librustdoc/html/render/mod.rs

```diff
@@ -2410,3 +2429,206 @@ fn collect_paths_for_type(first_ty: clean::Type, cache: &Cache) -> Vec<String> {
     }
     out
 }
+
+const MAX_FULL_EXAMPLES: usize = 5;
+
+/// Generates the HTML for example call locations generated via the --scrape-examples flag.
+fn render_call_locations(w: &mut Buffer, cx: &Context<'_>, def_id: DefId, item: &clean::Item) {
+    let tcx = cx.tcx();
+    let key = crate::scrape_examples::def_id_call_key(tcx, def_id);
+    let call_locations = match cx.shared.call_locations.get(&key) {
+        Some(call_locations) if call_locations.len() > 0 => call_locations,
```

Comment:
I have the same comment as when scraping - I think the `Option` is redundant, there should only be one way to represent "no call locations".

Reply:
Yes I think the `if call_locations.len() > 0` should be unnecessary now.